x/interchainstaking/keeper/receipt.go

```diff
@@ -114,43 +175,40 @@ func (k *Keeper) MintQAsset(ctx sdk.Context, sender sdk.AccAddress, senderAddres
 		qAssets = qAssets.Add(sdk.NewCoin(zone.LocalDenom, amount))
 	}
 
+	// check if a remote address exists for a non 118 coin type zone
+	setMappedAddress := true
+	if mappedAddress == nil && !zone.Is_118 {
+		var found bool
+		mappedAddress, found = k.GetRemoteAddressMap(ctx, sender, zone.ChainId)
+		if !found {
+			// if not found, skip minting and refund assets
+			return k.SendTokenIBC(ctx, k.AccountKeeper.GetModuleAddress(types.ModuleName), senderAddress, zone, assets[0])
```

Comment:
The assets in this case are still on the host zone, and should be sent using MsgSend from the zone.DepositAccount to the sender.